src/Chapter2.hs

```diff
@@ -500,7 +510,9 @@ True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
+isThird42 :: [Int] -> Bool
+isThird42 (_:_:42:xs) = True
```

Comment:
That's right!
Moreover, as you don't use `xs` in the body, you can also pattern match on it as on the wilcard:
```suggestion
isThird42 (_:_:42:_) = True
```